### PR TITLE
Add premproxy.com as additional scrapping site

### PR DESCRIPTION
**Pull request by Levonestral**
The code should work fine as is, but feel free to adjust as needed.

### CODE REVIEW

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: Instead of relying on having the second script with the `eval(function(p,a,c,k,e,d` maybe we could request all scripts and test if it starts with: `eval(function(p,a,c,k,e,d`. If yes, then proceed to unpack it. This is only for robustness and shouldn't be a priority.

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: `\(\'\.([\w\d]+)\'\).html\((\d+)\)` this regex pattern should pick up CSS classes and port numbers in one go without having to strip all other things. `$1` matches the CSS class and `$2` matches the port number.
Checkout: https://regexr.com/3nmbr
**Note**: I'm not sure if the pattern works directly in this case. Maybe another `re` method needs to be used.

This means that one could remove all the `unpack.replace()` calls.

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: You can decrement indentation of this last line (`return dictionary`) and remove the `return dictionary` from line 160.

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: IPs are going to be validated later on by `proxy_parser::__parse_proxylist()` so you can remove this check :)

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: Not sure if we want to grab "transparent" proxies because they're not anonymous...

File: proxytools/scrappers/premproxy.py
Comment (neskk) [inline proxytools/scrappers/premproxy.py]: Typo: `socks5` >> `HTTP`

**Review by neskk — CHANGES_REQUESTED** [review]
Thanks for contributing @Levonestral 👍
I have a few requests before I merge this, if you have any questions just ask.

**Comment by Levonestral** [comment]
Thanks for the review @neskk, I've made all the changes I think and a few minor adjustments also. Let me know if there is anything else needed.

**Review by neskk — APPROVED** [review]

**Pull request merged**
